estela-web/src/pages/JobDetailPage/index.tsx

```diff
@@ -660,6 +660,7 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
             spiderName,
             totalResponseBytes,
             items,
+            itemsCount,
```

Comment:
@joaquingx I think there's an issue here. We should obtain the `itemCount` in a similar manner as here: https://github.com/bitmakerla/estela/commit/3d39508ab0b30a30213c7f70dc0c40ff81e83eaf#diff-1763b2c090221df6b4323f653594f134a4a6ce3fd84f21e06140f5da2a66f08bR179

I have Redis stats set to update every 5 seconds, but the frontend shows a value on every page refresh, which doesn't make sense. The processing time doesn't update on every refresh, for example.

Reply:
I was getting item count in DB, I changed to get it from the redis, please review it.